modules/appeals_api/config/schemas/shared/v0/fileNumber.json

```diff
@@ -0,0 +1,13 @@
+{

```

Comment:
I added another shared schema for File Number because I found myself copying the description a few times